vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/bdd/steps/integration/ResourceCheckSteps.java

```diff
@@ -122,7 +121,7 @@ private void validateResources(Stream<WebPageResourceValidation> resourceValidat
 
     private WebPageResourceValidation validate(WebPageResourceValidation r)
     {
-        return r.getUri() == null || CheckStatus.FILTERED == r.getCheckStatus()
```

Comment:
`r.getUri() == null ` - why was it removed?

Reply:
uri == null if status is broken, imho it's better to read this piece of code by using status codes